### DISCUSSION TITLE
木马生成失败修复参考

### DISCUSSION THREAD

**Discussion by BOBOTSB** [sun589/QQkey_Tool#19] [提问] [answered]
[ERROR_2024-11-07_13.37.18.log](https://github.com/user-attachments/files/17656507/ERROR_2024-11-07_13.37.18.log)
这是那个日志 
![20ec57eb2e7080ada454d94a41c55062](https://github.com/user-attachments/assets/6d650adf-bfc7-4960-91ea-aae3ead8b15e)
还是显示这个

**Comment by sun589** [comment, marked as answer]
试下把这个压缩包里面的sd_pip_install脚本拖到程序目录之后打开安装再回去试试看看
~~[sd_pip_install.zip](https://github.com/user-attachments/files/17660591/sd_pip_install.zip)~~
新版:[lib_install_bat.zip](https://github.com/user-attachments/files/21680809/lib_install_bat.zip)
国内下载链接:https://wwap.lanzoum.com/iKOxp32zqopg

**Comment by BOBOTSB** [comment]
解决了

**Comment by BOBOTSB** [comment]
感谢！
